build_img.sh

```diff
@@ -141,6 +141,8 @@ function create_image () {
     dd if=output/imx-boot-sd.bin of=${PARTITION}1 bs=1K status=noxfer status=progress 2>/dev/null
```

Comment:
Commit message:
Please write a small paragraph about what has been changed and why